resolwe_bio/descriptors/default_clinical.yml

```diff
@@ -0,0 +1,1009 @@
+          required: false
+        - name: pah_score
+          label: PAH score
+          type: basic:decimal
+          required: false
+        - name: comments
+          label: Comments
+          type: basic:string
+          required: false
+    - name: molecular_markers
+      label: Molecular markers
+      group:
+        - name: p53_score
+          label: p53 score
+          type: basic:decimal
+          required: false
+        - name: proliferation_score
+          label: Proliferation score
+          type: basic:decimal
+          required: false
+        - name: vegfa_score
+          label: VEGFA score
+          type: basic:decimal
+          required: false
+        - name: vegfr_score
+          label: VEGFR score
+          type: basic:decimal
+          required: false
+        - name: kdr
+          label: KDR
+          type: basic:decimal
+          required: false
+        - name: itgb3
+          label: ITGB3
+          type: basic:decimal
+          required: false
+        - name: mlh1_loss_score
+          label: MLH1 loss score
+          type: basic:decimal
+          required: false
+        - name: hrd_loh
+          label: HRD-LOH
+          type: basic:decimal
+          required: false
+        - name: ctdna
+          label: ctDNA
+          type: basic:decimal
+          required: false
+        - name: comments
+          label: Comments
+          type: basic:string
+          required: false
+    - name: signatures
+      label: Signatures
+      group:
+        - name: immune_score
+          label: Immune score
+          type: basic:decimal
+          required: false
+        - name: mesenchymal_score
+          label: Mesenchymal score
+          type: basic:decimal
+          required: false
+        - name: msi_associated_gene_expressions
+          label: MSI associated gene expressions
+          type: basic:decimal
+          required: false
+        - name: stromal_score
+          label: Stromal score
+          type: basic:decimal
+          required: false
+        - name: tme_score
+          label: TME score
+          type: basic:decimal
+          required: false
+        - name: comments
+          label: Comments
+          type: basic:string
+          required: false
+    - name: public_or_private_database_data
+      label: Public or private database data
+      group:
+        - name: database_1
+          label: Database 1
+          type: basic:string
+          required: false
+        - name: database_2
+          label: Database 2
+          type: basic:string
+          required: false
+        - name: database_3
+          label: Database 3
+          type: basic:string
+          required: false
+        - name: database_4
+          label: Database 4
+          type: basic:string
+          required: false
+        - name: comments
+          label: Comments
+          type: basic:string
+          required: false
+    - name: onxerna_specific_prior_treatment
```

Comment:
If this is the default schema, shouldn't it be agnostic? Another option would be to rename it to something more specific.